### PR TITLE
Tweak All Settings page

### PR DESCRIPTION
**Pull request by rdwebdesign**
### What does this PR aim to accomplish?

1. Change the page title to **All Settings**, to match the string used on the menu.

2. The "All Settings" page is hidden in **_Basic_** and **_Advanced_** levels. To see the page we need to set the level to **_Expert_**.
  
    If we change the level to something else while the page is open, the page will be empty.

    empty_allsettings

### How does this PR accomplish the above?

Removing the level selector from this page fixes the issue.
This is fine because all information in this page is from "Expert" level.

### Link documentation PRs if any are needed to support this PR:

none

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
4. I have commented my proposed changes within the code and I have tested my changes.
5. I am willing to help maintain this change if there are issues with it later.
6. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
7. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
8. I have checked that another pull request for this purpose does not exist.
9. I have considered, and confirmed that this submission will be valuable to others.
10. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
11. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

### CODE REVIEW

**Comment by yubiuser** [comment]
Do you think we should also rename the file and the link from "advanced" to "all"?

**Comment by rdwebdesign** [comment]
I was thinking about that.

Maybe change the file name to `all.lp` or `allsettings.lp`.

**Comment by yubiuser** [comment]
I would keep the naming schema by `settings-XXX` > so this would be `settings-all.lp`

**Comment by DL6ER** [comment]
The naming schema is actually needed, FTL translates `-` to nested paths, so `/settings-all.lp -> /settings/all`

File: scripts/pi-hole/lua/settings_header.lp [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <!-- Title -->
 <div class="page-header flex-header">
     <h1><?= PageTitle ?></h1>
+<?
+if scriptname ~= 'settings/advanced' then
```
Comment (DL6ER) [inline scripts/pi-hole/lua/settings_header.lp]:
```suggestion
if scriptname ~= 'settings/all' then
```

**Review by DL6ER — CHANGES_REQUESTED** [review]

**Review by yubiuser — APPROVED** [review]
Screenshot at 2023-11-03 21-29-14

**Review by DL6ER — APPROVED** [review]

**Pull request merged**